stratum/hal/bin/barefoot/BUILD

```diff
@@ -56,3 +57,107 @@ stratum_cc_binary(
         "platforms/x86-64-inventec-d5254-r0.json"
     ],
 )
+
+# Rules for Debian package
+STRIP_PREFIX="/stratum/hal/bin/barefoot"
+
+pkg_tar(
+    name = "stratum_bf_bin",
+    srcs = [":stratum_bf"],
+    mode = "0755",
+    package_dir = "/usr/bin",
+    strip_prefix = STRIP_PREFIX,
+)
+
+pkg_tar(
+    name = "stratum_entry_point",
+    srcs = ["deb/stratum-entrypoint"],
+    mode = "0755",
+    package_dir = "/usr/bin",
+    strip_prefix = STRIP_PREFIX + "/deb",
+)
```

Comment:
I think these two rules can be combined. By default `pkg_tar` will flatten the directory hierarchy.